pkg/apis/traceflow/v1alpha1/doc.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020 Antrea Authors
```

Comment:
The directory should be "pkg/apis/ops/v1alpha1"

Reply:
Updated.